### PR TITLE
NN-4172: Check answers & prisoner report updates

### PR DESCRIPTION
**Pull request by natclamp-moj**

### CODE REVIEW

File: integration_tests/integration/ageOfPrisoner.spec.ts [hunk 1]
```diff
@@ -31,7 +31,6 @@ context('Age of the prisoner', () => {
               locationId: 234,
             },
             startedByUserId: 'TEST_GEN',
```
Comment (natclamp-moj) [inline integration_tests/integration/ageOfPrisoner.spec.ts:33]: don't need these here as this page is where that data is set

Comment (sp-watson) [reply integration_tests/integration/ageOfPrisoner.spec.ts:33]: yep!

File: integration_tests/integration/prisonerReport.spec.ts [hunk 2]
```diff
@@ -251,27 +292,33 @@ context('Prisoner report - reporter view', () => {
...
-            expect($summaryData.get(1).innerText).to.contain('Assaulting someone')
-            expect($summaryData.get(2).innerText).to.contain('Another prisoner - Paul Wright')
-            expect($summaryData.get(3).innerText).to.contain('Yes')
-            expect($summaryData.get(4).innerText).to.contain(
-              'Prison rule 51, paragraph 25(c)\n\nAssists another prisoner to commit, or to attempt to commit, any of the foregoing offences:\n\nPrison rule 51, paragraph 1\n\nCommits any assault'
-            )
+            expect($summaryData.get(2).innerText).to.contain('Assaulting someone')
+            expect($summaryData.get(3).innerText).to.contain('Another prisoner - Paul Wright')
+            expect($summaryData.get(4).innerText).to.contain('Yes')
+            // expect($summaryData.get(5).innerText).to.contain(
```
Comment (sp-watson) [inline integration_tests/integration/prisonerReport.spec.ts:319]: Do we need these comments?

File: integration_tests/integration/checkYourAnswersBeforeChangeReporter.spec.ts [hunk 3]
```diff
@@ -150,131 +91,302 @@ context('Check Your Answers', () => {
...
+        .find('dd')
+        .then($summaryData => {
+          expect($summaryData.get(0).innerText).to.contain('YOI offences\n\nPrison rule 55')
+          expect($summaryData.get(1).innerText).to.contain(
+            'Assault, fighting, or endangering the health or personal safety of others'
+          )
+          expect($summaryData.get(2).innerText).to.contain('Assaulting someone')
+          expect($summaryData.get(3).innerText).to.contain('Another prisoner - Paul Wright')
+          expect($summaryData.get(4).innerText).to.contain('Yes')
+          // expect($summaryData.get(5).innerText).to.contain(
```
Comment (sp-watson) [inline integration_tests/integration/checkYourAnswersBeforeChangeReporter.spec.ts:243]: Do we need these comments?

File: integration_tests/pages/ageofPrisonerSubmittedEdit.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import Page, { PageElement } from './page'
```
Comment (sp-watson) [inline integration_tests/pages/ageofPrisonerSubmittedEdit.ts:1]: Minor point - the name has a lower-case o.

File: server/views/macros/adjudicationSummary.njk [hunk 5]
```diff
@@ -15,14 +15,14 @@
...
      {% endif %}
     </div>
-      <div class="adjudicationsSummary-offenceDetails">
-        {% for offence in offences %}
-          {{ offenceQAndASummaryList(offence, { "data-qa": "offence-summary-table" }) }}
-        {% endfor %}
-      </div>
+    <div class="adjudicationsSummary-offenceDetails">
+      {% for offence in offences %}
+        {{ offenceQAndASummaryList(offence, incidentData.isYouthOffender, { "data-qa": "offence-summary-table" }) }}
```
Comment (sp-watson) [inline server/views/macros/adjudicationSummary.njk:23]: Ah good - we need this!

**Review by sp-watson — APPROVED** [review]

**Pull request merged**